src/cli/test_command.zig

```diff
@@ -1255,6 +1255,37 @@ export fn BunTest__shouldGenerateCodeCoverage(test_name_str: bun.String) callcon
     return true;
 }
 
+fn parseFileLineArg(arg: []const u8) ?struct { file_pattern: []const u8, line_num: u32 } {
+    // Find the last ':' to handle Windows drive letters like C:\foo\bar.test.ts:7
+    const colon_index = strings.lastIndexOfChar(arg, ':') orelse return null;
+    const after_colon = arg[colon_index + 1 ..];
+    if (after_colon.len == 0) return null;
+
+    // Check if there's a second colon (for file:line:column format)
+    const line_part = if (strings.indexOf(after_colon, ":")) |second_colon_index|
+        after_colon[0..second_colon_index]
+    else
+        after_colon;
+
+    const line_num = std.fmt.parseInt(u32, line_part, 10) catch return null;
```

Comment:
maybe it should try both if numbers, as if second one is a string, then use the last one again

Reply:
oh also there is no chance u can get another `:` after the lastIndexOf...?